lang/zig/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        ziglang zig 0.5.0
+github.tarball_from archive
+revision            0
+
+categories          lang
+platforms           darwin
+
+maintainers         userspace.com.au:felix openmaintainer
+
+description         robust, optimal, and maintainable programming language
+license             MIT
+long_description    A general-purpose programming language designed for \
+                    robustness, optimality, and maintainability.
+homepage            https://ziglang.org/
+
+checksums           rmd160  f6c957f66ebf9c981fea61cba4875081661ec3c6 \
+                    sha256  abe3e8ef064225f342bb24165e3c8a35f8f2c45930489a53897e835e984117eb \
+                    size    15743912
+
+
+depends_lib-append  port:llvm-9.0
+depends_build-append \
+                    port:clang-9.0
```

Comment:
We don't normally specify a compiler like this. Please see https://trac.macports.org/wiki/UsingTheRightCompiler to start. 

If you need a newer compiler, then you should use `compiler.blacklist` to prohibit the older ones.

Reply:
It looks like zig builds on too of the LLVM infrastructure to implement the new compiler, in which case specify a specific llvm/clang version is I think fine. Normally we would ask you to just blacklist those none not to work, but here I think it better to just specify macports clang 9.0 via the whitelist. With this you can remove clang from the depends_build list, but you should keep llvm setting in depends_lib as this is probably not done, as most ports do not need llvm at runtime.